emily/handler/src/database/entries/deposit.rs

```diff
@@ -28,6 +28,12 @@ pub struct DepositEntryKey {
     pub bitcoin_tx_output_index: u32,
 }
 
+impl std::fmt::Display for DepositEntryKey {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(f, "{}:{}", self.bitcoin_txid, self.bitcoin_tx_output_index)
```

Comment:
nit: I think `{}:{}` can be hard to parse by eyes. Maybe I'd prefer even `txid: {}, vout: {}`, even it looks bigger

Reply:
Would you mind if I punted on this one for some future PR?

Reply:
Actually, we can settle this in the current PR, and update the signer crate in a separate PR. But we should come to some consensus on the implementation. We have some choices though, with the most natural ones being:
1. `txid:{}:vout:{}` (or `{}:{}`)
2. `txid:{};vout:{}` (or `{};{}`)
3. `txid:{},vout:{}` (or `{},{}`)
4. `txid:{} vout:{}` (or `{} {}`)
5. `txid:{}, vout:{}` (or `{}, {}`)

We could also remove the `Display` implementation if none of these jump out. I'm leaning towards remove the implementation.

Reply:
I'd vote for option 2. About doing it in another PR it is ok for me and let's not block on this